### ISSUE TITLE
Add Service Provider Autoloading for Laravel 5.5

### ISSUE THREAD

**Issue by jagDanJu** [Webklex/laravel-git-hook#3]
Add this to the composer.jso to autoload den Service Provider. There ist no need to add this manually on Laravel 5.5 :)

```
"extra": {
    "laravel": {
        "providers": [
            "Webklex\\GitHook\\Providers\\LaravelServiceProvider“
        ]
    }
}
```

**Comment by Webklex** [comment]
Hi @jagDanJu ,
thanks for your sugestion but this repository supports all Laravel versions from 5.0.0 on. So by adding this I would breake older systems..

**Comment by jagDanJu** [comment]
@Webklex I dont think that you would break older systems .. < 5.5 this will be ignored. Than the User needs to add the Service Provider manually

**Comment by Webklex** [comment]
@jagDanJu well that would be greate. I'll run a little test and release a new version soon :)

Best regards
